build.psm1

```diff
@@ -172,8 +172,10 @@ function Start-BuildNativeWindowsBinaries {
         [ValidateSet('Debug', 'Release')]
         [string]$Configuration = 'Release',
 
-        [ValidateSet('x64', 'x86')]
-        [string]$Arch = 'x64'
+        [ValidateSet('x64', 'x86', 'x64_arm64', 'x64_arm')]
```

Comment:
`x64_arm` and `x64_arm64` seems weird to me.  Why are we combining two architectures?

Reply:
I'll add a comment.  This is the form that the Visual Studio build environment expects for cross compilation.  The first is the host and the second the target.  VS supports x86_arm64, for example which I'm not supporting here.

Reply:
@SteveL-MSFT Can you comment?